station.go

```diff
@@ -462,7 +462,7 @@ func (c *Conn) removeFunctionsUpdatesListener(stationName string) error {
 
 	sfs, ok := c.stationFunctionSubs[sn]
 	if !ok {
-		return memphisError(errors.New("functions listener doesn't exist"))
+		return nil
```

Comment:
all changes in this file seem like an issue but you return nil, could you please explain?